wodles/aws/aws_s3.py

```diff
@@ -74,6 +74,14 @@ def main(argv):
                 bucket_type = buckets_s3.server_access.AWSServerAccess
             else:
                 raise Exception("Invalid type of bucket")
+            if options.regions:
+                for invalid_region in options.regions:
+                    try:
+                        if invalid_region not in aws_tools.ALL_REGIONS:
+                            raise ValueError(f"Invalid region '{invalid_region}'")
```

Comment:
Using `invalid_region` is a bit confusing, I would suggest renaming the variable. The regions should be considered valid unless the validation fails.

```suggestion
                for region in options.regions:
                    try:
                        if region not in aws_tools.ALL_REGIONS:
                            raise ValueError(f"Invalid region '{region}'")
```

Reply:
The variable name was changed to avoid confusion.